app/models/user_role.rb

```diff
@@ -30,8 +30,16 @@ def self.facility_management_roles
     facility_roles - [FACILITY_STAFF, FACILITY_SENIOR_STAFF]
   end
 
+  def self.global_roles
+    account_manager + administrator + billing_administrator
+  end
+
   def self.valid_roles
-    account_manager + administrator + billing_administrator + facility_roles
+    global_roles + facility_roles
+  end
+
+  def self.global
+    where(facility_id: nil)
```

Comment:
Would this be better as `where(role: global_roles)`? I just know there are some weird cases in UIC where there is an `Administrator` role with an `facility_id`. I believe these were added by hand, but it seems like the role name should take precedence over `facility_id: nil`. I could be swayed by a good argument, though.